tests/setupTests.js

```diff
@@ -50,6 +50,7 @@ beforeAll(async () => {
     hostname: TEST_LOCATION,
     replace: jest.fn()
   };
+  window.navigator.userLanguage = 'en';
```

Comment:
that most likely won't work, as ?most of? the navigator attributes are immutable... but you should be able to overwrite them just like the `window.location` and `.sessionStorage` right above 👍 (and that should also make the snapshots pass again)